scipy/stats/_continuous_distns.py

```diff
@@ -3975,7 +3975,8 @@ class johnsonsb_gen(rv_continuous):
 
         f(x, a, b) = \frac{b}{x(1-x)}  \phi(a + b \log \frac{x}{1-x} )
 
-    for :math:`0 <= x < =1` and :math:`a, b > 0`, and :math:`\phi` is the normal
+    for all :math:`(x, a, b) \in \mathbb{R}^3` where
+    :math:`0 <= x < =1` and :math:`b > 0`, and :math:`\phi` is the normal
```

Comment:
```suggestion
    :math:`0 \leq x \leq 1` and :math:`b > 0`, and :math:`\phi` is the normal
```

Reply:
Thank you for the suggestion. agreed and commit in cec2ef9f634e0c02865e1e7fb32beb7aca2fed45